### DISCUSSION TITLE
Urgent Please help

### DISCUSSION THREAD

**Discussion by Khaos1221** [lllyasviel/Fooocus#2375] [Q&A]
Guys my browser said COnnnection timed out and I refreshed I know the CLI is still working , Now I have lost the webpage of the prompt, Can Someone please share How to bring it back?? I dont want to build a complete image as My pc is not able to handle the requests pretty well

OR is there any way of looking for current Iteration?

**Comment by ILikeToasters** [comment]
http://127.0.0.1:7865/ is the default address if running locally. You can also just shut down the CMD window and start again.

**Comment by mashb1t** [comment]
The prompt is in the terminal output, please chrck it and find it there.

**Comment by Khaos1221** [comment]
Guys once you close the page the only way You can see it is in output at the end thank you guys anyways
